crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -768,6 +771,52 @@ impl<'w, 's> Commands<'w, 's> {
     ) -> EntityCommands {
         self.spawn(Observer::new(observer))
     }
+
+    /// Registers a system and returns a [`SystemId`] so it can later be called by [`World::run_system`].
+    ///
+    /// It's possible to register the same systems more than once, they'll be stored separately.
```

Comment:
```suggestion
    /// It's possible to register the same systems more than once; they'll be stored separately.
```

Small grammar nit :)